magento/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "bitnami/magento:{{ .Values.imageTag }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
+        - name: MARIADB_HOST
+          value: {{ template "mariadb.fullname" . }}
+        - name: MARIADB_PORT
+          value: "3306"
+        - name: MARIADB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "mariadb.fullname" . }}
+              key: mariadb-root-password
+        - name: MAGENTO_HOST
+          value: {{ default .Values.magentoLoadBalancerIP .Values.magentoHost | quote }}
```

Comment:
The linter returns the following linter warnings.

We need to think what to do in these cases.

```
helm lint .
==> Linting .
[WARNING] templates/deployment.yaml: these substitution functions are returning no value: [{{ default .Values.magentoLoadBalancerIP .Values.magentoHost | quote }}]
[WARNING] templates/svc.yaml: these substitution functions are returning no value: [{{ default "" .Values.magentoLoadBalancerIP }}]

1 chart(s) linted, no failures
```